src/transformers/models/phi/convert_phi_weights_to_hf.py

```diff
@@ -172,4 +186,11 @@ def convert_phi_weights(checkpoint_path, pytorch_dump_folder_path, use_cuda, sav
     )
 
     args = parser.parse_args()
-    convert_phi_weights(args.checkpoint_path, args.pytorch_dump_folder_path, args.use_cuda, args.save_weights_directly)
+    convert_phi_weights(
+        args.model_name,
+        args.checkpoint_path,
+        args.pytorch_dump_folder_path,
+        args.use_cuda,
+        args.save_weights_directly,
+        _MODELS,
```

Comment:
should not need to be passed as it's a global mapping

Reply:
Hmm yes but if I remove this, I am getting referenced before assignment error...